public/translations/en-US.json

```diff
@@ -883,11 +895,72 @@
     "success": "Settings saved successfully",
     "community_info": {
       "title": "Community Information",
-      "description": "Logo, name, description"
+      "description": "Logo, name, description, url",
+      "guidance": "Guidance",
+      "logo": {
+        "title": "Community logo",
+        "description": "Your file must be in PNG, JPG or SVG format. It should measure 80px in width and 80px in height to be better viewed on the interface."
+      },
+      "invitation": {
+        "title": "Access to the community",
+        "description": "When a community \"invitation\" is set, new members only get access through an invitation."
```

Comment:
I wonder if this could be more well phrased. Maybe something like:

"When a community is invite only" or something like this. The way it is looks like a long round trip to get to this meaning